subsys/logging/Kconfig.formatting

```diff
@@ -192,6 +192,11 @@ config LOG_OUTPUT_FORMAT_DATE_TIMESTAMP
 	help
 	  When enabled timestamp is formatted to YYYY-MM-DD hh:mm:ss.ms,us.
 
+config LOG_OUTPUT_FORMAT_ISO8601_TIMESTAMP
+	bool "Format timestamp in date format"
```

Comment:
This description needs to be reworked, or else you would have two options with same short descriptions (see the one above).

Reply:
Sure